client/app/views/templates/view_steps_agreement.jade

```diff
@@ -2,4 +2,7 @@ extends view_base
 
 block region
     h1=t('Conditions générales')
```

Comment:
Maybe we could do this during markup integration ? Until now those labels are purely indicative.